### ISSUE TITLE
No reason for startup notifications

### ISSUE THREAD

**Issue by Hirbod** [arnesson/cordova-plugin-firebase#664]
I don't see any reason for this lines. They could be removed instantly, as they weren't implemented in the previous versions either. grantPermission() can be called on "deviceready" if the user wan't an immediate Permission window. This is a breaking change for old users and should be reverted asap @soumak77 

https://github.com/arnesson/cordova-plugin-firebase/blob/c6e8067f9ae3f68de14b129fcc5d3fa7da1f9234/src/ios/AppDelegate%2BFirebasePlugin.m#L47-L80

**Comment by soumak77** [comment]
Desired functionality is mentioned here: https://github.com/arnesson/cordova-plugin-firebase/issues/648#issuecomment-382526917

**Comment by Hirbod** [comment]
Yeah @soumak77 and it's useless.. As said, just call grantPermission() on deviceready. Instead going the old way, you introduced a BC without even mentioning it anywhere

**Comment by soumak77** [comment]
I only recently started maintaining this plugin.  Seems this change came from https://github.com/arnesson/cordova-plugin-firebase/pull/576 which was merged by @robertarnesson.

@Hirbod so this is not needed to fix iOS 11 as the PR indicates?  If that is the case it's fine to remove the code

**Comment by Hirbod** [comment]
I did not had any issues with iOS 11, even with version 0.1.25.
I just deleted out the lines shown above and notifications are working, grantPermission is working and onNotificationOpen also. I just tested this on iOS 10 and iOS 11, as my app don't has support for prior versions.

IMHO this PR (at least those lines) aren't useful. But I don't know what the other lines are doing.. maybe it fixed something for @nekator 

Maybe @nekator could say something. Is it safe to remove the above mentioned lines? In my cases, everything worked as expected.

**Comment by Hirbod** [comment]
As mentioned in his PR-title, he actually just wanted to implemented the "didReceiveMessage", but due to lack of real code reviewing, evereything else also got merged.

It should be safe to remove the lines above.

grantPermission is enough on deviceready
https://github.com/arnesson/cordova-plugin-firebase/blob/c6e8067f9ae3f68de14b129fcc5d3fa7da1f9234/src/ios/FirebasePlugin.m#L93-L111

**Comment by nekator** [comment]
@Hirbod

You are right it should be safe to replace the code with a grantPermission call. To be honest I can't remember why I implemented it that way... my only goal was to implememt didReceiveMessage.

I'm sorry that this change  broke the plugin for others.

**Comment by Hirbod** [comment]
Thanks for clarification @nekator, I will provide a PR
